packages/testcontainers/__tests__/containers/RegTestContainer/index.test.ts

```diff
@@ -37,5 +37,11 @@ describe('regtest', () => {
       const address = await container.getNewAddress('not-default', 'legacy')
       expect(address.length).toStrictEqual(34)
     })
+
+    it('should be able to getnewaddress with label and as eth', async () => {
+      const address = await container.getNewAddress('not-default', 'eth')
+      expect(address).toMatch(/^0x[a-fA-F0-9]{40}$/gm)
```

Comment:
Based on the conversation below, can we use ethers is address? I think it's a better checker

Reply:
we don't have ether related libs integration yet in jellyfish.. the regex is actually good to assert the eth style address

Reply:
I think eventually we will need to add this. Thoughts? @fuxingloh

Reply:
yes we will need this especially for signing a evm tx... either adapter ethersjs or straight away import it like bignumberjs.. haven't come out a full picture on tx builder part yet

Reply:
should we just add a todo here so we can come back later once ethers is added?

Reply:
hmm.. properly we don't need ethersjs.. as rpc `eth_*` will be supported on defi-cli.. 
https://github.com/DeFiCh/ain/pull/2029
(note that `eth_sign` in pending)

if we need ethersjs.. will add it in other PR

i think i'll add a note pointing to reference for easier
https://github.com/ethers-io/ethers.js/blob/5210b68a7837654c6b84207a45e1e573d9472d1a/src.ts/address/address.ts#L123

Reply:
Yeah, let's explore it in another PR to keep the scope small for this.